modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/ModelUtils.java

```diff
@@ -741,6 +741,17 @@ public static boolean isURISchema(Schema schema) {
                 && URI_FORMAT.equals(schema.getFormat());
     }
 
+    private static boolean isEnumSchema(final Schema<?> schema) {
+        // MyEnum:
+        //   type: string
+        //   enum:
+        //   - ENUM_1
+        //   - ENUM_2
+        return SchemaTypeUtil.STRING_TYPE.equals(schema.getType()) // type: string
```

Comment:
what about enum of integers?

Reply:
You are right. It has to be for all types of enum. I change it